subprojects/packagefiles/icu/source/data/meson.build

```diff
@@ -10,7 +10,14 @@ endif
 
 if host_machine.system() == 'windows'
   if cpp.get_argument_syntax() == 'msvc'
```

Comment:
Same as previous.

Reply:
Here it's because I'm checking for MSVC vs MinGW. I've moved the check to the GCC/Clang clause, leaving this one for MSVC only.